runtime/etc/Makefile.mli-common

```diff
@@ -20,21 +20,32 @@
 # Makefile.mli-common
 #
 
+# Only include Makefile.include when we are compiling the server. Otherwise,
+# we'll build pieces of the client with the target compiler. Note that if
```

Comment:
Somewhat picky, but you might slightly rephrase the "Note" here. First, "if" implies a hypothetical but really it's an actuality: we _don't_ include `Makefile.include` for the client. So maybe say "Note that since ..." instead of "Note that if ...". The, with respect to "the client only needs access to" at the end, maybe say  something generic like "a few selected files" rather than naming `chpllaunch.chpl`. Naming that one file creates a dependence on the code which is unlikely to get updated here if and when the code does change, making this comment misleading. Being vague about exactly which files are involved will push anyone who needs to know this to go look at the code to find out.

Reply:
Not picky at all, comments are also part of the code! :)